src/commands/tests/mod.rs

```diff
@@ -589,15 +599,30 @@ async fn do_test_on_package(
                 )
                 .await;
             }
-            let (stdout, stderr, success) = execute_command(
-                &fslabs_test.command,
-                &package_path,
-                &fslabs_test.envs,
-                &HashSet::new(),
-                Some(tracing::Level::DEBUG),
-                Some(tracing::Level::DEBUG),
-            )
-            .await;
+            let (stdout, stderr, success) = match fslabs_test.id == "cargo_lock" {
```

Comment:
I don't get why this is a special case when the test has a command: `fslabscli fix-lock-files --check`. Are we even using that command?

Reply:
That's a good point, I hadn't. thought of actually calling fslabscli from within fslabscli.

Reply:
Rereading the code, the `fslabscli fix-lock-files --check` command can be run locally if you want, hence it's existence.

This command would run for the whole repo.

In the prow tests we actually wants to run, not at the repo level, but at the workspace level, and only for those that changed, that's why it's a specific case.